java-symbolic-execution/java-symbolic-execution-plugin/src/test/files/se/UnclosedResourcesLombokCheck.java

```diff
@@ -0,0 +1,18 @@
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import lombok.Cleanup;
+
+class UnclosedResourcesLombokCheck {
+  public void missingAnnotation(String fileName) throws IOException {
+    InputStream in = new FileInputStream(fileName); // Noncompliant
+    in.read();
+  }
```

Comment:
We can replace this part of the sample with a fully qualified lombok annotation so that this file only contains cases where  we should not raise
```suggestion
  public void fullyQualified(String fileName) throws IOException {
    @lombok.Cleanup
    InputStream in = new FileInputStream(fileName);
    in.read();
  }
```